tests/index.ts

```diff
@@ -623,6 +623,7 @@ describe('Requests', () => {
       const params = (myRequest as any).params;
       expect(params.params).toEqual({});
       expect(params.headers).not.toBeUndefined();
+      expect(params).not.toHaveProperty('data');
```

Comment:
I'm not sure these tests make sense... params _is_ data, no?

Reply:
So params is the config object given to Axios. Data would be `params.data` in this case.
For POST requests this would look like:
```
{  
    url: myPath,
    headers: combinedHeaders,
    data: options.data
}
```

Because Axios uses a different key for sending data on a GET request we were changing this to:
```
{  
    url: myPath,
    headers: combinedHeaders,
    data: options.data,
    params: { data: option.data }
}
```

With RN and Axios having anything in the original data parameter causes the request to get stuck in promises and execute when you close the app :jackie:. So this PR change changes the object to:
```
{  
    url: myPath,
    headers: combinedHeaders,
    params: { data: option.data }
}
```

Reply:
Hella confusing that the params are at `.params.params`. XD

Reply:
param inception 🤣